pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Do we need this change?

Reply:
you can merge this, pnpm did something weird and updated this to 6.1 but it should be 6.0 👌